chamber/utils/logging.py

```diff
@@ -29,8 +29,7 @@ def emit(self, record):
             for k, v in record.__dict__.items()
             if k not in self.DEFAULT_STREAM_HANDLER_VARIABLE_KEYS.union(self.CUSTOM_STREAM_HANDLER_VARIABLE_KEYS)
         }
-        record.msg = '{} --- {}'.format(record.msg, json.dumps(extra, cls=DjangoJSONEncoder,
-                                        default=lambda x: '<<NON-SERIALIZABLE TYPE: {}>>'.format(type(x).__qualname__)))
+        record.msg = '{} --- {}'.format(record.msg, json.dumps(extra, cls=DjangoJSONEncoder))
```

Comment:
1. Why are you removing this?
2. What's gonna happen if serialization fails? It will throw an exception, right? Is that expected behavior?

Reply:
1. It does not work as expected
2. Yes and that's expected behavior